src/m365/spo/commands/tenant/tenant-recyclebinitem-remove.ts

```diff
@@ -69,58 +68,8 @@ class SpoTenantRecycleBinItemRemoveCommand extends SpoCommand {
   }
 
   public async commandAction(logger: Logger, args: CommandArgs): Promise<void> {
-    const removeDeletedSite = async (): Promise<void> => {
-      try {
-        this.spoAdminUrl = await spo.getSpoAdminUrl(logger, this.debug);
-        const res: FormDigestInfo = await spo.ensureFormDigest(this.spoAdminUrl, logger, this.context, this.debug);
-        if (this.verbose) {
-          logger.logToStderr(`Removing deleted site collection ${args.options.siteUrl}...`);
-        }
-
-        const requestOptions: any = {
-          url: `${this.spoAdminUrl as string}/_vti_bin/client.svc/ProcessQuery`,
-          headers: {
-            'X-RequestDigest': res.FormDigestValue
-          },
-          data: `<Request AddExpandoFieldTypeSuffix="true" SchemaVersion="15.0.0.0" LibraryVersion="16.0.0.0" ApplicationName="${config.applicationName}" xmlns="http://schemas.microsoft.com/sharepoint/clientquery/2009"><Actions><ObjectPath Id="16" ObjectPathId="15" /><Query Id="17" ObjectPathId="15"><Query SelectAllProperties="false"><Properties><Property Name="PollingInterval" ScalarProperty="true" /><Property Name="IsComplete" ScalarProperty="true" /></Properties></Query></Query></Actions><ObjectPaths><Method Id="15" ParentId="1" Name="RemoveDeletedSite"><Parameters><Parameter Type="String">${formatting.escapeXml(args.options.siteUrl)}</Parameter></Parameters></Method><Constructor Id="1" TypeId="{268004ae-ef6b-4e9b-8425-127220d84719}" /></ObjectPaths></Request>`
-        };
-
-        const processQuery: string = await request.post(requestOptions);
-        const json: ClientSvcResponse = JSON.parse(processQuery);
-        const response: ClientSvcResponseContents = json[0];
-        if (response.ErrorInfo) {
-          throw response.ErrorInfo.ErrorMessage;
-        }
-        else {
```

Comment:
We don't need this else statement. That will save some indentation

Reply:
That if/else statement is removed in this first commit [33fbfb592406030828702482822a192c24f692a1](https://github.com/pnp/cli-microsoft365/pull/5131/commits/33fbfb592406030828702482822a192c24f692a1)?